docs/content/get-started/upload-launch.md

```diff
@@ -35,53 +35,60 @@ Your `spatialos.json` should look like this:
 }
 ```
 
+You also need to set your project name in the `Deployments` window of the Unity Editor. Select `SpatialOS > Deployment Launcher` to open then `Deployments` window.
+
+Fill in the "Project Name" field at the top of the window in the Unity Editor.
+
 <br/>
 ### Upload worker assemblies
 
 An [assembly](https://docs.improbable.io/reference/latest/shared/glossary#assembly) is a bundle of code, art assets and other files necessary to run your game in the cloud.
 
-To run a deployment in the cloud, you must upload the worker assemblies to your SpatialOS project. You can only do this in a terminal, via the [spatial CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli). You must also give the worker assemblies a name so that you can reference them when launching a deployment.
+To run a deployment in the cloud, you must upload the worker assemblies to your SpatialOS project. Do this using the `Deployments` window in the Unity Editor. Select `SpatialOS > Deployment Launcher` to open then `Deployments` window.
 
-Do this using a terminal of your choice - for example, PowerShell on Windows - navigate to `gdk-for-unity-fps-starter-project/` and run `spatial cloud upload <assembly_name>`. The `<assembly_name>` is a string you choose made up of alphanumeric characters, `_`, `.`, and `-`; for example `my_assembly`. A valid upload command would look like this:
+To upload an assembly, under the "Assembly" section, fill in the `Assembly Name` field in the window and select Upload Assembly. The `Assembly Name` is a string you choose made up of alphanumeric characters, `_`, `.`, and `-`; for example `my_assembly`
 
+A valid configuration looks like the following:
 
-```
-spatial cloud upload my_assembly
-```
+<img src="{{assetRoot}}assets/deployment-window.png" style="margin: 0 auto; display:block;" />
 
-<br/>
-<br/>
-**It’s finished uploading when:** You see an upload report printed in your terminal output, for example:
+> **It’s finished uploading when:** You see an upload report printed in your Unity console, for example:
 ```
-Upload report:
-- 5 artifacts uploaded (4 successful, 1 skipped, 0 failed)
+Uploaded assembly my_assembly to project unity_gdk successfully.
 ```
 <br/>
 Based on your network speed, this may take a little while (1-10 minutes) to complete.
 
 <br/>
+
 ### Launch a cloud deployment
```

Comment:
Looking at the rendered page (I am looking at the page on your branch), I think we need to rejig this whole page slightly, as the tasks we are doing are all in the Deployments window. Listing it the old way makes it much more complex.
It's impossible for me to show you want I am suggesting for a rejig it on a PR.
I know you hate GDocs - what's the best way to do this? PR to your branch? Thoughts?

Reply:
PR please